app/src/main/java/org/fossasia/pslab/fragment/LALogicLinesFragment.java

```diff
@@ -181,7 +181,24 @@ private void updateLogicLines() {
             }
             LineDataSet lineDataSet = new LineDataSet(tempInput, channelNames.get(j));
             lineDataSet.setCircleRadius(1);
-            lineDataSet.setColor(Color.GREEN);
+            switch (j) {
+                case 0:
+                    lineDataSet.setColor(Color.BLUE);
```

Comment:
This color is very hard to see in a black background. Always think of a user's point of view.